nifi-frontend/src/main/frontend/apps/nifi/src/app/pages/flow-designer/ui/canvas/items/processor/edit-processor/edit-processor.component.ts

```diff
@@ -296,6 +321,21 @@ export class EditProcessor extends TabbedDialog {
         return this.nifiCommon.formatBundle(entity.component.bundle);
     }
 
+    formatRunStatus() {
+        if (this.status.runStatus === 'Stopped' && this.status.aggregateSnapshot.activeThreadCount > 0) {
+            return `Stopping (${this.status.aggregateSnapshot.activeThreadCount})`;
+        }
+
+        return `${this.status.runStatus}`;
+    }
+
+    getValidationErrorsTipData(entity: any): ValidationErrorsTipInput {
+        return {
+            isValidating: entity.status.validationStatus === 'VALIDATING',
+            validationErrors: entity.component.validationErrors
+        };
+    }
```

Comment:
Seems like now that we have introduced validation status and tooltip that it quickly becomes confusing to the user if they make a change that makes it valid but the yield icon and tooltip are stale.
A quick example would be to add a Generate Flow File processor. initially it is invalid since it does not have a connection. This is good and the tooltip reflects this. However, if i auto-terminate that connection the validation isn't reprocessed and it seems like their might still be an issue.

This could be a big can of worms.

Reply:
Removed validation errors tooltip.